tests/foreman/cli/test_myaccount.py

```diff
@@ -165,23 +159,64 @@ def test_negative_update_email(self):
         for email in invalid_emails_list():
             with self.subTest(email):
                 with self.assertRaises(CLIReturnCodeError):
-                    User.update({
-                        'login': self.user['login'],
-                        'mail': email,
-                    })
+                    self.update_user(
+                        {'login': self.user['login'], 'mail': email}
+                    )
 
-    @stubbed()
     @tier1
-    def test_negative_update_password_invalid(self):
-        """Update My Account with invalid Password/Verify fields
+    def test_negative_update_locale(self):
+        """Update My Account with invalid Locale
 
-        @id: f9230699-fb8e-45d6-a0c2-abb8b751304d
-
-        @Steps:
-        1. Update Current user with all variations of Password/Verify fields
-        in [2]
+        @id: 6f63c9b4-80e6-11e6-8ea1-68f72889dc7f
 
         @Assert: User is not updated. Appropriate error shown.
+        """
+        with self.assertRaises(CLIReturnCodeError):
+            self.update_user({'locale': 'invalid'})
 
-        @caseautomation: notautomated
+
+class MyAccountEphemeralUserTestCase(CLITestCase):
```

Comment:
Performance test: 

setupClass version: 4m51s

```console
============================= test session starts ==============================
platform linux2 -- Python 2.7.12, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: /home/renzo/PycharmProjects/robottelo, inifile: 
plugins: cov-2.3.0, xdist-1.14
collected 9 items

test_myaccount.py 2016-09-23 09:06:14 - robottelo - DEBUG - Started setUpClass: tests.foreman.cli.test_myaccount/MyAccountTestCase
2016-09-23 09:06:14 - robottelo.cli.factory - DEBUG - User "2YZKoW" password not provided

.2016-09-23 09:11:05 - robottelo - DEBUG - Started tearDownClass: tests.foreman.cli.test_myaccount/MyAccountEphemeralUserTestCase


========================== 9 passed in 292.38 seconds ==========================

Process finished with exit code 0
``` 
setup version : 7m 24s
```console
============================= test session starts ==============================
platform linux2 -- Python 2.7.12, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: /home/renzo/PycharmProjects/robottelo, inifile: 
plugins: cov-2.3.0, xdist-1.14
collected 9 items

test_myaccount.py 2016-09-23 09:17:47 - robottelo - DEBUG - Started setUpClass: tests.foreman.cli.test_myaccount/MyAccountTestCase
2016-09-23 09:17:47 - robottelo - DEBUG - Started Test: MyAccountTestCase/test_negative_update_email

.2016-09-23 09:25:11 - robottelo - DEBUG - Started tearDownClass: tests.foreman.cli.test_myaccount/MyAccountTestCase


========================== 9 passed in 445.02 seconds ==========================

Process finished with exit code 0
```

I know in our overall running its nothing, but I am not sure the code is so difficult to understand. Checking a third opinion: @rochacbruno ;)